package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Project page: http://js-intro.burlingtoncodeacademy.com/projects/tic-tac-toe-www",
   "main": "main.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "npx static-server -p 8080"
```

Comment:
You could also say `"npx node-static ."` since node-static is already set up to run on port 8080, but good use of your scripts to make starting the server easier.